app/actions/actionTypes.tsx

```diff
@@ -142,6 +142,7 @@ export enum ACTION_TYPES {
   PAPER_SHOW_FAILED_TO_GET_OTHER_PAPERS_FROM_AUTHOR = "PAPER_SHOW_FAILED_TO_GET_OTHER_PAPERS_FROM_AUTHOR",
   PAPER_SHOW_START_TO_GET_COLLECTIONS = "PAPER_SHOW_START_TO_GET_COLLECTIONS",
   PAPER_SHOW_SUCCEEDED_GET_COLLECTIONS = "PAPER_SHOW_SUCCEEDED_GET_COLLECTIONS",
+  PAPER_SHOW_START_TO_GET_COLLECTIONS_IN_DROPDOWN = "PAPER_SHOW_START_TO_GET_COLLECTIONS_IN_DROPDOWN",
```

Comment:
I think of that you don't need to make this Action and flag.
is there any problem to use `PAPER_SHOW_START_TO_GET_COLLECTIONS` action to make loading flag?

Reply:
`PAPER_SHOW_START_TO_GET_COLLECTIONS` is already change isLoadingCollection state in reducer. 

The problem is that the right side collection note section of the paper show is rendered together as the isLoadingCollection trigger changes. Even if no memo data, the height of the right section changes repeatedly each time the collection drop down list is opened.

To prevent it and render only the collection drop down list, the corresponding action was added.